src/components/LimitedWidthContainer.tsx

```diff
@@ -0,0 +1,17 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+
+export const limitedWidthContainerStyle = css`
+  --max-inner-width: calc(1440px - var(--sidenav-collapsed-width) - calc(2 * var(--global-horizontal-padding)));
+
+  max-width: var(--max-inner-width);
+  position: relative;
+  margin: 0 auto;
+`
+export const LimitedWidthContainer = styled.div`
+  --max-inner-width: calc(1440px - var(--sidenav-collapsed-width) - calc(2 * var(--global-horizontal-padding)));
+
+  max-width: var(--max-inner-width);
+  position: relative;
+  margin: 0 auto;
+`
```

Comment:
```suggestion
export const LimitedWidthContainer = styled.div`
  ${limitedWidthContainerStyle}
`
```
Let's not repeat yourself. Also, I'm generally okay with it, but not sure why you've changed the name.

Reply:
Oh - ok. I know why - because we are applying it to Studio and Viewer views.